lib/cyclops/RPC_defs.h

```diff
@@ -1,13 +1,80 @@
 /** @file RPC_defs.h
     @brief Defines a few constants of the RPC Format Specification.
            This file is part of CL.
-  
-    @section header-desc HEADER Field Description
-    ~~~~~~~~~~~~~~~~~~~~
-        <2: channel> <6: command>
-        [7:6] Channel
-        [5:0] Command
-    ~~~~~~~~~~~~~~~~~~~~
+    
+    @page rpc-desc RPC Format Description
+
+    @tableofcontents
+
+    @section rpc-overview Overview
+    There are two kinds of Serial packets, single byte and multi byte.
+    The size of a packet is determined by MSB of header and the ``command`` field.
+
+    @section S-header-desc Single Byte Packets
+    Single byte Headers are distinguished from other headers by the MSB bit. For
+    Single byte headers, this bit is always set
+    
+    Field       | Bits  | Description
+    ----------- | ----- | -----------
+    Reserved    | [7]   | Always ``1``.
+    ``channel`` | [6:3] | Determines if "command" is applied on the Channel @par 1: Command is appplied on Channel
+    ``command`` | [2:0] | The command field
+    
+    @subsection S-cmd-desc Command descriptions
+    
+    command[3:0] | Name     | Effect
+    ------------ | -------  | -------------
+    ``0000``     | start    | Launch mainloop
+    ``0001``     | stop     | Pause mainloop
+    ``0010``     | reset    | Reset all sources. @attention The system is *not* reset to _initial configuration_!
+    ``0011``     | swap     | Swap the Cyclops instances of the 2 high ``channel`` bits.
```

Comment:
We need 4 bits for the command, 4 bits for the channel (since we need to specify 2) and the 7th bit to indicate that this is single byte header. But this adds up to 9 bits.